index.html

```diff
@@ -4,215 +4,230 @@
+                                      </div>
+                                    </li>
+                                     <li>
+                                    <div class="tech-skills-detail">
+                                      <img src="images/Ellipsehtml.png" alt="HTML">
+                                      <p>HTML</p>
+                                    </div>
+                                 </li>
+                            
                                 <li>
-                                    <a class="icons" href="#"><img src="images/Vector(3).svg" alt="icons"></a>
-                                </li>
-                                <li>
-                                    <a class="icons" href="#"><img src="images/Vector.svg" alt="icons"></a>
-                                </li>
-                                <li>
-                                    <a class="icons" href="#"><img src="images/Vector(1).svg" alt="icons"></a>
+                                  <div class="tech-skills-detail">
+                                     <img src="images/Ellipsecss.png" alt="CSS">
+                                     <p>CSS</p></div>
                                 </li>
+                                </ul>
+                            </li>
+                            <li class="ok"><div class="flex1"><h2>Frameworks</h2> <i class="fas fa-angle-right"></i></div></li>
+                            <li class="ok"><div class="flex1"><h2>Skills</h2> <i class="fas fa-angle-right"></i></div></li>
+    
                         </ul>
-                    </nav>
-                    <div> 
-                        <button class="bull" type="submit">Get my resume</button> 
+    
                     </div>
-            </section>
-
-            <section class="lang">
-                    <div class="langu"> 
-                    <div class="frame"><h2>Languages</h2><i class="fas fa-angle-down"></i></div>
-                    <div class="program">
-                        <ul class="pro">
-                            <li class="js"><img src="images/Ellipsejs.png" alt="icons"> javascript</li>
-                            <li class="js"><img src="images/Ellipsehtml.png" alt="icons"> HTML</li>
-                            <li class="js"><img src="images/Ellipsecss.png" alt="icons">CSS</li>
-                    </ul>
-                    <ul class="fra">
-                        <li class="fro"><h2>Frameworks </h2><i class="fas fa-angle-right"></i></li>
-                        <li class="fro"><h2>Skills </h2><i class="fas fa-angle-right"></i></li>
-                    </ul>
-                </div>
-                </div>
-            </section>
-        </section> 
-     </main>
-    </body>
-     </html>
-     
\ No newline at end of file
+                </section>
+                                <div class="flex-container"> 
+                <section class="contactform1">
+                    <h1>Contact me</h1>
```

Comment:
- [ ] [REQUIRED] Please, make sure to always use only one `h1` element in the whole document for better SEO Ranking. You can check for "semantic h1 uses`

Reply:
corrected it thanks